controllers/userController.js

```diff
@@ -0,0 +1,71 @@
+
+import * as UserModel from "../models/userModel";
```

Comment:
Why are you importing 8 functions when you only use 3. Please import only functions that you will use.